README.md

```diff
@@ -129,6 +189,8 @@ deployment.apps/sonataflow-platform-jobs-service condition met
 backstage.rhdh.redhat.com/backstage condition met
 deployment.apps/backstage-backstage condition met
 ```
+If a deployment failure occurs for a Custom Resource (CR), check the logs of the pods created by the corresponding job responsible for deploying the failed CR instance.
```

Comment:
What is the point here?
Are you referring to CRD instead of CR? Because the only Job that the chart creates are the `...crd-availability-check` to check the availability of some CRDs and the corresponding `...-cleanup` to remove them

Reply:
This sentence is already part of main branch (see [here](https://github.com/parodos-dev/orchestrator-helm-chart/blob/2847a3baa8ec7cc8b74815b07ab2082822350abc/README.md?plain=1#L116)). Reworded. Let me know if it's clear